huxley/api/urls.py

```diff
@@ -27,6 +27,7 @@
     url(r'^schools/?$', views.school.SchoolList.as_view(), name='school_list'),
     url(r'^schools/(?P<pk>[0-9]+)/?$', views.school.SchoolDetail.as_view(), name='school_detail'),
     url(r'^schools/(?P<pk>[0-9]+)/assignments/?$', views.school.SchoolAssignments.as_view(), name='school_assignments'),
+    url(r'^schools/(?P<pk>[0-9]+)/assignments/finalize', views.school.SchoolAssignmentsFinalize.as_view(), name='school_assignments_finalize'),
```

Comment:
I think this indicates that you might have an outdated branch. Make sure that 'fab update' runs in the 'fab submit' script so you get the latest version of the branch.